docs/reference/getting-started.asciidoc

```diff
@@ -1092,7 +1092,7 @@ In SQL, the above aggregation is similar in concept to:
 
 [source,sh]
 --------------------------------------------------
-SELECT state, COUNT(*) FROM bank GROUP BY state ORDER BY COUNT(*) DESC
+SELECT bank.`state.keyword`, COUNT(*) FROM bank GROUP BY bank.`state.keyword` ORDER BY COUNT(*) DESC LIMIT 10
```

Comment:
I think adding the limit is a good additing, and I see why you want to add the keyword to the target column, but I think it makes the example too complicated for something that should only be "similar in concept". Maybe it would be better to add a short note to the example above explaining that "keyword" refers to an unanalyzed version of the "state" field.